File: Практическая./hash.cpp
#include <iostream>
#include <unordered_map>

int main() {
    // Создаем хеш-таблицу
    std::unordered_map<std::string, int> myMap;
    
    // Добавляем элементы
    myMap["apple"] = 3;
    myMap["banana"] = 2;
    myMap["orange"] = 1;
    
    // Выводим все элементы
    for (const auto& element : myMap) {
        std::cout << element.first << " = " << element.second << std::endl;
    }
    
    return 0;
}

File: hash.java
import java.util.HashMap;

public class SimpleExample {
    public static void main(String[] args) {
        // Создаем хеш-таблицу
        HashMap<String, Integer> myMap = new HashMap<>();
        
        // Добавляем элементы
        myMap.put("apple", 10);
        myMap.put("banana", 20);
        myMap.put("orange", 15);
        
        // Выводим все элементы
        for (String fruit : myMap.keySet()) {
            System.out.println(fruit + ": " + myMap.get(fruit));
        }
    }
}
